### PR TITLE
ci: [SDKCF-2712] change ci config to use circle ci orbs (android-sdk) for build and publish

### PR DESCRIPTION
**Pull request by vincentdelmundo**
## Links
https://jira.rakuten-it.com/jira/browse/SDKCF-2712

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -119,15 +24,33 @@ workflows:
...
               only: /^v.*/
             branches:
               ignore: /.*/
-      - publish:
+      - android-sdk/publish:
           requires:
             - release-verification
+          post-steps:
+            - run:
+                name: Publish Documentation
```
Comment (corycaywood) [inline .circleci/config.yml:38]: I've also added a config in `android-builconfig` for automatically publishing documentation when `./gradlew publish` is run: https://github.com/rakutentech/android-buildconfig/tree/master/publish/documentation#setup

But probably you can add that sometime later.

**Review by corycaywood — APPROVED** [review]
Looks good 👍

File: .circleci/config.yml (hunk 1 above)
Comment (vincentdelmundo) [reply .circleci/config.yml:38]: Cool. I'll create separate PR for that.
Thanks

**Review by MdUsmanGani — APPROVED** [review]
LGTM

**Pull request merged**